packages/desktop-client/src/components/reports/reports/CashFlowCard.tsx

```diff
@@ -127,6 +339,10 @@ export function CashFlowCard({
           : '/reports/cash-flow'
       }
       menuItems={[
+        {
+          name: 'changeView',
+          text: t('Change view'),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
          name: 'change-view',
          text: someVariable ? t('Switch to condensed graph') : t('Switch to detailed graph'),
```